<launch>
  <node name="iologik_dio" pkg="iologik_dio" type="iologik_dio" output="screen">
    <param name="rate" value="10" />

    <remap from="/input0" to="/dio/in/wd_ok" />
    <remap from="/input1" to="/dio/in/ts_btn_ok" />
    <remap from="/input2" to="/dio/in/ecu_ok" />
    <remap from="/input3" to="/dio/in/wd_assert" />
    <remap from="/input4" to="/dio/in/imd_ok" />
    <remap from="/input5" to="/dio/in/bypass_status" />
    <remap from="/input6" to="/dio/in/dv_btn" />
    <remap from="/input7" to="/dio/in/asms_status" />
    <remap from="/input8" to="/dio/in/sdc_out" />

    <remap from="/output0" to="/dio/out/arm_ebs" />
    <remap from="/output1" to="/dio/out/arm_dbs" />
    <remap from="/output2" to="/dio/out/wd_trigger" />
    <remap from="/output3" to="/dio/out/ts_btn" />
    <remap from="/output4" to="/dio/out/sdc_close" />
    <remap from="/output5" to="/dio/out/wd_reset" />

  </node>
</launch>